### PR TITLE
Run doctests with tox (and py.test)

### PR DESCRIPTION
**Pull request by redapple**

### CODE REVIEW

**Comment by redapple** [comment]
@kmike , are you ok with this?

**Comment by kmike** [comment]
👍

File: conftest.py [hunk 1]
```diff
@@ -1,7 +1 @@
-import six
-collect_ignore = []
-
-if six.PY3:
-    for fn in open('tests/py3-ignores.txt'):
```
Comment (eliasdorneles) [inline conftest.py]: did this fix all PY3 tests?
should the py3-ignores.txt file be removed?

Comment (redapple) [inline conftest.py]: ah, damn, I knew I had forgotten something. I was meant to remove that file ealier.
Only `urljoin_rfc` doctest fails with Python 3 (as it's deprecated, I didn't dig and skipped it)

**Pull request merged**